src/test/java/com/google/sps/servlets/IntervieweeFeedbackServletTest.java

```diff
@@ -59,7 +57,6 @@
   private FakeScheduledInterviewDao scheduledInterviewDao;
   private FakePersonDao personDao;
   private FakeEmailSender emailSender;
-  private FakeEmailUtils emailUtils;
```

Comment:
How come we got rid of the fake?

Reply:
EmailUtils is now a regular class that has static methods that both the tests and real code can use